pages/docs/development.mdx

```diff
@@ -193,13 +193,13 @@ The `@grouparoo/demo` plugin makes it easy to generate sample data! You can use
 ```bash
 cd apps/staging-community
 # populate the system with 1000 profiles, properties, 1000 purchases, groups, and some events
-(npx) grouparoo demo-data-purchases
+grouparoo demo-data-purchases
 
 # including the --scale param allows you to control how many profiles you make. e.g. --scale 10 makes 10,000 extra profiles.
-(npx) grouparoo demo-data-purchases --scale 10
+grouparoo demo-data-purchases --scale 10
 
 # populate the system with events
-(npx) grouparoo demo-event-stream
```

Comment:
I think this is OK for now... but for development, we should figure out a away use the local (`../../cli/dist/grouparoo.js`) more easily